src/qml/qgismobileapp.qml

```diff
@@ -189,8 +214,8 @@ ApplicationWindow {
     CoordinateLocator {
       id: coordinateLocator
       anchors.fill: parent
-      visible: stateMachine.state === "digitize"
-      highlightColor: digitizingToolbar.isDigitizing ? digitizingRubberband.color : "#CFD8DC"
+      visible: stateMachine.state === "digitize" || stateMachine.state === 'measure'
+      highlightColor: digitizingToolbar.isDigitizing ? stateMachine.state === "digitize" ? digitizingRubberband.color : measuringRubberband.color : "#CFD8DC"
```

Comment:
it might be clearer to have a variable like `currentRubberand: stateMachine.state === "digitize" ? digitizingRubberband: measuringRubberband` and use the `currentRubberand` everywhere.